src/luascript.cpp

```diff
@@ -12608,6 +13360,383 @@ int LuaScriptInterface::luaPartySetSharedExperience(lua_State* L)
 	return 1;
 }
 
+// Loot
+
+int LuaScriptInterface::luaCreateLoot(lua_State* L)
+{
+	// Loot() will create a new loot item
+	Loot* loot = new Loot();
```

Comment:
tested several cases now, it seems like relaod already clears the memory as they're bound to lua it seems that the GC takes care of it already, atleast I had no unusual memory increase, memory stayed still for 5 minutes even tho I kept reloading monsters (the entire time) with tons of test files :D